### PR TITLE
[DOCS] Update alias API docs for data stream aliases

### PR DESCRIPTION
**Pull request by jrodewig**
Updates the docs for several alias APIs for data stream aliases. Also
removes some redundant examples now covered in the [aliases guide][0].

[0]: https://www.elastic.co/guide/en/elasticsearch/reference/master/alias.html

### Previews
* Aliases guide: https://elasticsearch_73419.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/alias.html#add-alias
* Create/update alias API: https://elasticsearch_73419.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/indices-add-alias.html
* Alias exists API: https://elasticsearch_73419.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/indices-alias-exists.html
* Aliases API: https://elasticsearch_73419.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/indices-aliases.html
* Delete alias API: https://elasticsearch_73419.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/indices-delete-alias.html
* Get alias API: https://elasticsearch_73419.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/indices-get-alias.html

### CODE REVIEW

**Review by martijnvg — APPROVED** [review]
LGTM, thanks James!

File: docs/reference/indices/aliases.asciidoc [hunk 1]
```diff
@@ -30,138 +35,124 @@ POST /_aliases
...
-(Optional, Boolean)
-If `true`, assigns the index as an alias's write index.
-Defaults to `false`.
-+
-An alias can have one write index at a time.
+// tag::alias-options[]
+`index_routing`::
+(Optional, string) Value used to route indexing operations to a specific shard.
+If specified, this overwrites the `routing` value for indexing operations. Data
+stream aliases don't support this parameter.
```
Comment (martijnvg) [inline docs/reference/indices/aliases.asciidoc:114]: This line can now be removed. I just merged #73462.

Comment (jrodewig) [reply docs/reference/indices/aliases.asciidoc:114]: Thanks. I've removed the line for the `is_write_index` parameter.

**Comment by jrodewig** [comment]
Thanks @martijnvg.

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Pull request merged**